src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/get_indices.js

```diff
@@ -16,6 +16,11 @@ export async function getIndices(es, rawPattern, limit) {
     return [];
   }
 
+  // ES does not like just a `,*` and will throw a `[string_index_out_of_bounds_exception] String index out of range: 0`
+  if (pattern.startsWith(',')) {
+    return [];
+  }
```

Comment:
Wondering if there should be some feedback to the user about this case.  Might be more of a pain than it's worth for how much of an edge case this is.

Reply:
Great question. I thought about this and in my mind, this query wouldn't match anything in the first place so showing no results without additional messaging felt right to me, but open to changing if others feel strongly

Reply:
I do not feel strongly either way.  I can imagine maybe that someone types a comma by mistake and doesn't notice it.  Not sure how realistic that is.